db/migrate/20170406161501_create_forms.rb

```diff
@@ -0,0 +1,12 @@
+class CreateForms < ActiveRecord::Migration[5.0]
+  def change
+    create_table :forms do |t|
+      t.string :description, null: false
+      t.string :url, null: false
+      t.boolean :active, null: false, default: false
+      t.string :created_by, null: false
```

Comment:
This is for the user that created it but I wanted it readable and was unsure how to do that easily with the current User model.